src/Sulu/Bundle/ContactBundle/Resources/public/js/services/account-delete-dialog.js

```diff
@@ -144,7 +144,7 @@ define([
 
     return {
         showDialog: function(ids, okCallback) {
```

Comment:
@nnatter add docs for the dialog methods. Especially if you allow to pass arrrays and integers